pkg/kubelet/cm/cpumanager/options.go

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
Can we call this file `policy_options.go`

Reply:
ack